applications/StructuralMechanicsApplication/tests/cpp_tests/test_solid_transient_sensitivity.cpp

```diff
@@ -134,7 +134,7 @@ SolvingStrategyType::Pointer CreatePrimalSolvingStrategy(ModelPart* pModelPart)
     SchemeType::Pointer p_scheme =
         Kratos::make_shared<ResidualBasedBossakDisplacementScheme<SparseSpaceType, LocalSpaceType>>(0.0);
     ConvergenceCriteriaType::Pointer p_conv_criteria =
-        Kratos::make_shared<ResidualCriteria<SparseSpaceType, LocalSpaceType>>(1e-24, 1e-25);
+        Kratos::make_shared<ResidualCriteria<SparseSpaceType, LocalSpaceType>>(1e-12, 1e-14);
```

Comment:
@loumalouomega Could this cause a difference in the execution time?

Reply:
All the contrary, the number of iterations will decrease. In fact that was the origin of the origin PR. That the cpp tests did not converge due to a tight convergence tolerance

Reply:
Then I have no clue.

Reply:
Sincerely I don't know